app/build.gradle

```diff
@@ -37,11 +37,13 @@ android {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+            buildConfigField "String", "FRONTEND_URL", '"https://eventyay.com/e/"'
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
             buildConfigField "String", "STRIPE_API_KEY", '"'+STRIPE_API_TOKEN+'"'
         }
         debug {
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
+            buildConfigField "String", "FRONTEND_URL", '"https://open-event-fe.netlify.com/e/"'
```

Comment:
It's fossasia.github.io/open-event-frontend

Reply:
I take it from the frontend documentation. Will update it. :+1:

Reply:
Done